addons/binding/org.openhab.binding.max/src/main/java/org/openhab/binding/max/internal/handler/MaxCubeBridgeHandler.java

```diff
@@ -129,7 +130,11 @@ public MaxCubeBridgeHandler(Bridge br) {
     private boolean propertiesSet = false;
```

Comment:
I'm pretty sure they don't, the only one is maybe about `@NonNull` marked fields (I'm interested in a reference for that). It's valid practice to leave them out, only local variables require initilaization. Note also that a lot of fields are already not initialised via the constructor. 

While checking the code again, I saw something which troubles me more :-)

The constructor is above the fields, can you move it below the fields.